scripts/datasheet-rendering/render-datasheets.sh

```diff
@@ -6,17 +6,30 @@ if [ -n "$CI" ]; then
     # Fix for SSL problem in phantom.js
     # export OPENSSL_CONF=/etc/ssl/
 
+    fc-list :lang=zh
+
+    apt-get install -y language-pack-zh*
```

Comment:
I'd check if this is even needed.

Reply:
It seems to work without these lines, thank you!